docs/DeveloperGuide.md

```diff
@@ -111,40 +114,80 @@ low|user adopting this products|clear all my contacts from the address book|clea
 3. User requests to delete a specific schedule in the list.
 4. Teaching Assistant deletes the schedule.
 
-    Use case ends.
+   Use case ends.
 
 **Extensions**
 
 * 2a. The list is empty.
 
-    Use case ends.
+  Use case ends.
 
 * 3a. The given schedule name is invalid.
 
-  * 3a1. Teaching Assistant shows an error message.
+    * 3a1. Teaching Assistant shows an error message.
 
-    Use case ends.
+      Use case ends.
 
-<br>
+---
 
-### Non-Functional Requirements
+## Non-Functional Requirements
 
 1. Should work on any mainstream OS as long as it has Java 11 or above installed.
 2. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
 3. The system should be usable by a novice who has never used virtual management applications.
 4. The user interface should be intuitive enough for users who are not IT-savvy.
 5. The product is offered as an open source software.
 
-### Glossary
+---
 
-**JC**<br>
+## Glossary
+
+### JC
 Junior College (JC) is the post-secondary education level where students are preparing for university.
 JC is also the high-school equivalent in other countries. Hence, JC teachers may be packed with consultation
 schedules which can leverage our software.
 
-**Mainstream OS**<br>
+### Mainstream OS
 Mainstream operating systems are the current operating systems with a significant market share, namely Windows, Linux,
 Unix, and OS-X.
 
-**MSS**<br>
+### MSS
 Main Success Scenario (MSS) defines the optimal outcome of our commands, i.e. in the case where no errors occurred.
+
+---
+
+## Implementation
```

Comment:
Perhaps this 'Implementation' section can be shifted up so that it's in a new section rather than under 'Appendix: Requirements'?